polynote-frontend/style/styles.less

```diff
@@ -1098,7 +1098,7 @@ body {
         }
       }
 
-      .output[mime-type='text/plain'] {
+      .output[mime-type*='text/'] {
```

Comment:
Is this style still necessary? I'm thinking `text/html` would match here, but probably shouldn't be displayed like preformatted text.

Reply:
Ah, agreed this is no longer needed! I'll revert this change